### ISSUE TITLE
Auto Rename Functions

### ISSUE THREAD

**Issue by KleinMandolin** [BryanFrederickson/CYBER1-CodeVersion#16]
**As a** dev
**I need** a part of the semantic clone generation to change the functions
**So that** the output is a different signature.

### Details and Assumptions
* will be done as a preliminary test for trying out libCST and manipulating the code

### Acceptance Criteria
```gherkin
Given the source code is valid
When a function is encountered in the tree
Then the function name will be changed
```

**Comment by KleinMandolin** [comment]
Function Def renaming has been implemented as of Commit 7d3321b, but not for corresponding function calls throughout tree.

**Comment by KleinMandolin** [comment]
Automatic function def renaming has been fully implemented as of Commit b55bd40. All custom function names defined in definition or via import alias, as well as function calls for custom functions, are renamed and preserve code functionality.
